### PR TITLE
feat: text query

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: internal/query/message_text.go [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+	if !ok {
+		contents, err = q.readTranslationFile(q.NotificationDir, fmt.Sprintf("/i18n/%s.yaml", language))
+		if errors.IsNotFound(err) {
+			contents, err = q.readTranslationFile(q.NotificationDir, fmt.Sprintf("/i18n/%s.yaml", q.DefaultLanguage.String()))
+		}
+		if err != nil {
+			return nil, err
+		}
+		q.NotificationTranslationFileContents[language] = contents
+	}
```
Comment (livio-a) [inline internal/query/message_text.go:187]: reading from file is exactly the same as above, maybe put it in a separate func

File: internal/query/message_text.go [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+	result := notificationText.GetMessageTextByType(messageType)
+	result.IsDefault = true
+	return result, nil
+}
+
+func (q *Queries) OrgMessageTextByTypeAndLanguage(ctx context.Context, messageType, language string) (*MessageText, error) {
+	//Merged Texts
+	return nil, nil
+}
```
Comment (livio-a) [inline internal/query/message_text.go:220]: ?

File: internal/query/mail_template.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	}
+	MailTemplateColTemplate = Column{
+		name: projection.MailTemplateTemplateCol,
+	}
+	MailTemplateColIsDefault = Column{
+		name: projection.MailTemplateIsDefaultCol,
+	}
+	MailTemplateColState = Column{
+		name: projection.MailTemplateStateCol,
+	}
```
Comment (livio-a) [inline internal/query/mail_template.go:50]: please add table to columns

**Review by livio-a — CHANGES_REQUESTED** [review]
please add tests for queries

File: internal/query/message_text.go (hunk 2 above)
Comment (fgerschwiler) [reply internal/query/message_text.go:220]: Bot used

**Review by livio-a — CHANGES_REQUESTED** [review]
could you please merge in `main` and there are some fields which could be null but this is not handled

**Review by livio-a — APPROVED** [review]

**Pull request merged**